### PR TITLE
[iOS] Fix touch conflict between Shell Flyout and SwipeView (on Device)

### LINKED ISSUE #9088: [Bug] SwipeView items conflict with Shell menu swipe in from left, on real iOS devices
### Description
Combining Shell with SwipeView items in a CollectionView works in iOS Simulator, but not on a real iOS device.

Specifically, swipe right conflict with swiping the menu in from the left.

### Steps to Reproduce
Please see the link to the repro. The repro includes both SwipeItems in a CollectionView and a BoxView using the `SwipeGestureRecognizer`. The BoxView with the `SwipeGestureRecognizer` seem to work fine.

Notice that it is sometimes possible to swipe right on a SwipeItem. As far as I can tell swiping precisely on on of the `Label` texts helps, but even this does not seem consistent. 

### Expected Behavior
On Android there is no such observed issue. On android Shell menu swipe in from the left and SwipeView items in a CollectionView does not conflict.

### Basic Information

- Version with issue: 4.4.0.991265
- Last known good version: N/A
- IDE: VS 16.4.2, .NET Core runtime 3.1.0
- Platform Target Frameworks: 
  - iOS:  MDK 6.6.0.155, Xamarin.iOS 13.8.3.0, Xcode 11.3(15712)
- Nuget Packages:  Xamarin.Forms 4.4.0.991265, Xamarin.Essentials 1.3.1
- Affected Devices: iOS devices, but not the iOS Simulator

### Reproduction Link
https://github.com/1iveowl/XF.ShellSwipe.Repro1

### PR DESCRIPTION
**Pull request by 1iveowl**
### Description of Change ###

Fix touch conflict between Shell Flyout and SwipeView (on Device).

### Issues Resolved ### 
- fixes #9088 

### API Changes ###
None

### Platforms Affected ### 
- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
See provided UITest

### PR Checklist ###

- [x] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by 1iveowl** [comment]
@jsuarezruiz 

From the repro and UITest I can see that it is currently the speed of the swipe that determines if the swipe is being picked up by the Shell Menu or by the SwipeView. 

Slower swipes are picked up by the SwipeView, while faster swipes are picked up by the Shell Menu. 
 
I’ve looked at the iOS SwipeView Renderer, but I cannot identify a root cause from this. The issue is too tricky for my current understanding of how Shell and SwipeView is constructued. 

I was hoping that you might have some insights here? 

Thanks

**Comment by jsuarezruiz** [comment]
@1iveowl Thank you very much for your PR. With the repro sample it has been easier to fix the issue. I have applied changes to your PR to add the fix and have verified it in simulator and devices.

File: Xamarin.Forms.sln.DotSettings [hunk 1]
```diff
@@ -669,6 +669,7 @@
 	<s:Double x:Key="/Default/Environment/InjectedLayers/InjectedLayerCustomization/=File64B48C7709A839499C5E49DBCE502A37/RelativePriority/@EntryValue">1</s:Double>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpKeepExistingMigration/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpPlaceEmbeddedOnSameLineMigration/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpUseContinuousIndentInsideBracesMigration/@EntryIndexedValue">True</s:Boolean>
```
Comment (jsuarezruiz) [inline Xamarin.Forms.sln.DotSettings]: @1iveowl  Could you undo this not necessary change?. If you want, let me know, I can do it.

Comment (1iveowl) [reply Xamarin.Forms.sln.DotSettings]: I'm only on a phone at the moment. Would be great if you can just undo it. Thx

**Comment by jsuarezruiz** [comment]
@samhouts Should we retarget this to 4.5?

**Comment by samhouts** [comment]
@1iveowl Are you able to rebase to 4.6.0? Thanks!

**Comment by rmarinho** [comment]
@1iveowl can you rebase? thanks

**Pull request merged**